message_ix_models/tools/costs/projections.py

```diff
@@ -0,0 +1,777 @@
+from itertools import product
+
+import numpy as np
+import pandas as pd
+
+from message_ix_models.tools.costs.config import (
+    BASE_YEAR,
+    FIRST_MODEL_YEAR,
+    HORIZON_END,
+    HORIZON_START,
+)
+from message_ix_models.tools.costs.gdp import adjust_cost_ratios_with_gdp
+from message_ix_models.tools.costs.learning import (
+    project_ref_region_inv_costs_using_learning_rates,
+)
+from message_ix_models.tools.costs.regional_differentiation import (
+    apply_regional_differentiation,
+)
+from message_ix_models.tools.costs.splines import apply_splines_to_convergence
+
+
+class projections:
+    def __init__(self, inv_cost, fix_cost):
+        self.inv_cost = inv_cost
+        self.fix_cost = fix_cost
+
+
+def smaller_than(sequence, value):
```

Comment:
According to coverage/pytest-cov/Codecov this function and `larger_than()` are not tested, and according to jedi/VS Code they are not used anywhere in the module. Safe to remove?